src/desktop/apps/home/client/index.coffee

```diff
@@ -28,25 +27,6 @@ module.exports.HomeView = class HomeView extends Backbone.View
     new HomeAuthRouter
     Backbone.history.start pushState: true
 
-    if !sd.CURRENT_USER
-      isExperiment = sd.HOMEPAGE_COLLECTION_HUB_ENTRYPOINTS_TEST is "experiment"
-      targetElement = if isExperiment then ".home-hubs-entry" else ".home-browse-module"
-      # Remove after closing the homepage hubs entry points test
-      $(targetElement).waypoint(() ->
```

Comment:
A question for @abhitip - Is it useful to keep this impression post-experiment?

Reply:
That's a great point Steve! I don't think we need the impression event anymore :)

Reply:
I'm going to ago ahead and merge this.